src/test/java/com/premiumminds/webapp/utils/ApplicationContextConfigurationTest.java

```diff
@@ -18,43 +18,45 @@
  */
 package com.premiumminds.webapp.utils;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
 import java.io.InputStream;
 import java.net.URL;
 
 import org.eclipse.jetty.testing.ServletTester;
 
-import junit.framework.TestCase;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
-public class ApplicationContextConfigurationTest extends TestCase {
+public class ApplicationContextConfigurationTest {
 	private ServletTester tester;
 	
-	@Override
-	protected void setUp() throws Exception {
+	@BeforeEach
+	protected void setUp() {
 		tester = new ServletTester();
 		
 		URL url = this.getClass().getResource("/");
 		tester.setResourceBase(url.getPath());
 	}
-	
-	@Override
-	protected void tearDown() throws Exception {
-	}
-	
+
+	@Test
 	public void testSuccessLoad(){
 		tester.addEventListener(new ApplicationContextConfigurationListener());
 		tester.getContext().setInitParameter("app-config", "classpath:/com/premiumminds/webapp/utils/applicationConfigurationTest.properties");
 		try {
 			tester.start();
 			assertTrue(ApplicationContextConfiguration.get().containsKey("application"));
-			assertTrue("test".equals(ApplicationContextConfiguration.get().getProperty("application")));
+			assertEquals(ApplicationContextConfiguration.get().getProperty("application"), "test");
```

Comment:
the expected value should be in the left side, right?

Reply:
You are right. 

Damn you intellij recommendations!

changed.